utils/proxy/forwarder.py

```diff
@@ -28,7 +28,7 @@ def __init__(self):
 
         # This is the proxy state. Basically, true or false values that tells the proxy to enable, or not
         # specific behavior. You can get/modify it using a direct GET/POST /_system_tests_state request to the proxy
-        self.state = json.loads(os.environ.get("INITIAL_PROXY_STATE", "") or "{}")
+        self.state = json.loads(os.environ.get("INITIAL_PROXY_STATE", "") or '{"mock_remote_config_backend": true}')
```

Comment:
I ended up with timing problems if I tried to set the state within the test class. I need the first RC update request from the tracer on the /v0.7/config endpoint to be given a mocked response. This worked for now but if you have a better less hacky way to do so please let me know. 😄